vuepress/docs/.vuepress/next.js

```diff
@@ -121,188 +173,232 @@ module.exports = {
+                            },
+                            {
+                                title: 'Install Plugins and Micro Frontends without a Bundle',
+                                path: path + 'tu-create-components/pb/deploy-components-without-bundle.md'
+                            }
+                        ]
+                    }
+                    
                 ]
             },
             {
-                title: 'Bundles',
+                title: 'Curate and Share Applications',
                 children: [
-                    path  + 'ecr/publish-simple-bundle.md',
-                    path  + 'ecr/publish-project-bundle.md',
-                    path  + 'ecr/export-bundle-from-application.md',
-                    path  + 'ecr/github-actions-workflow.md',
-                    path  + 'ecr/deploy-components-without-bundle.md',
-                    path  + 'ecr/ecr-private-git-repo.md',
-                    path  + 'ecr/ecr-private-images.md',
-
+                    path  + 'tu-curate-share/ecr-private-git-repo.md',
+                    path  + 'tu-curate-share/ecr-private-images.md',
                 ]
             },
             {
-                title: 'Extend the Platform',
+                title: 'Compose an Application',
                 children: [
-                    {
-                        title: 'Extend App Builder',
-                        path: path  + 'customize-the-platform/extend-app-builder.md',
-                    },
-                    {
-                        title: 'Add REST API',
-                        path: path  + 'customize-the-platform/add-rest-api.md',
-                    },
-                    {
-                        title: 'Change Default Datasource',
-                        path: path  + 'customize-the-platform/change-default-datasources-and-connections/',
-                    },
+                    path  + 'tu-compose-app/page-management.md',
+                    path  + 'tu-compose-app/widgets-fragments.md',
+                    path  + 'tu-compose-app/content-tutorial.md',
+                    path  + 'tu-compose-app/content-types-tutorial.md',
+                    path  + 'tu-compose-app/content-attributes.md',
+                    path  + 'tu-compose-app/content-templates-tutorial.md',
+                    path  + 'tu-compose-app/digital-assets-tutorial.md',
+                    path  + 'tu-compose-app/creating-protected-resources.md',
+                    path  + 'tu-compose-app/extend-app-builder.md'
                 ]
             },
             {
-                title: 'Configuration and Operations',
+                title: 'Configure and Customize an Application',
                 children: [
                     {
                         title: 'Default Database',
-                        path: path + 'devops/default-database.md',
+                        path: path + 'configure-customize/default-database.md',
```

Comment:
And shorten "configure-customize", maybe just back to "devops". Even though we don't use the term in the nav, it's more descriptive in a word for what is included here.